test/http_body_test.dart

```diff
@@ -116,7 +116,8 @@ void _testHttpServerRequestBody() {
       }
       body.request.response.close();
     }, onError: (error) {
-      if (!shouldFail) throw error;
+      // ignore: only_throw_errors
```

Comment:
cc @natebosch suggestions? I don't like ignoring things but this is a false positive really.